src/containers/studies/StudiesSagas.js

```diff
@@ -142,6 +145,50 @@ const LOG = new Logger('StudiesSagas');
 const CAFE_ORGANIZATION_ID = '7349c446-2acc-4d14-b2a9-a13be39cff93';
 const TIME_USE_DIARY = 'Time Use Diary';
 
+function* deleteStudyWorker(action :SequenceAction) :Saga<*> {
```

Comment:
No change required. You should start putting sagas (and reducers) into their own files like we do with the orgs app.